dll/win32/shell32/folders/CDrivesFolder.cpp

```diff
@@ -1138,7 +1138,7 @@ HRESULT WINAPI CDrivesFolder::GetDefaultColumnState(UINT iColumn, DWORD * pcsFla
 
```

Comment:
Change pcsFlags type from DWORD* to SHCOLSTATEF* ?

Reply:
Changed. Thanks.